src/main/java/Display.java

```diff
@@ -0,0 +1,48 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class Display {
+    public void printGreeting() {
+        System.out.println("Welcome to TicTacToe!");
+    }
+
+    public void promptPlayer(String playerMark) {
+        System.out.println(String.format("Make a move: %s", playerMark));
+    }
+
+    private List currentGridCells(Board board, int size) {
+        List currentGridAsArray = new ArrayList<>();
+        Board boardClass = board;
+        int gridSize = size;
+        int count = 1;
+
+        while (count <= gridSize) {
```

Comment:
This looks like a for loop, could you take a go and simplifying this by changing the while to a for?

Reply:
✅ changed to `for` PR #6